src/cmd/embeddedrules/rules.php

```diff
@@ -9,6 +9,7 @@
  * @comment Report ternary expressions that can be simplified.
  * @before  $x ? $x : $y
  * @after   $x ?: $y
+ * @fix     $x ?: $y
```

Comment:
`$x` should be pure. You can't remove duplicated expression if it may have side effects.